src/courses/templates/courses/chart_xp_progress.html

```diff
@@ -1,7 +1,3 @@
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js"></script>
-<!-- https://github.com/chartjs/Chart.Zoom.js/blob/master/samples/zoom.html -->
-<!-- <script type="text/javascript" src="https://raw.githubusercontent.com/chartjs/Chart.Zoom.js/master/Chart.Zoom.min.js"></script></script> -->
-
 <script type="text/javascript">
```

Comment:
This should be a .js file too

Reply:
cant do it since template tags doesnt seem to work with static files